logexp/logexp.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script when run on a node (/master) in a kuberentes cluster would
+# export the logs to gcs, with the upload path and other config taken
+# through environment variables.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+# The following env vars need to be defined before calling this script:
```

Comment:
I would prefer we pass in arguments rather than rely on environment variables. Any chance on making this change?

Also thoughts on writing this in a language other than bash (python or golang?)